tests/main/remove-auto-connections/task.yaml

```diff
@@ -0,0 +1,43 @@
+summary: Automatic connection gets dropped on refresh
+
+details: |
+      Automatic connection gets removed and not reported by connections if respective plug
+      is no longer present in the refreshed snap.
+
+prepare: |
+  snap install jq
+  snap pack test-snap-v1
+  snap pack test-snap-v2
+
```

Comment:
If you make some changes to the pull request would you mind adding this fragment:
```
restore: |
    snap remove simplesnap
    rm -f simplesnap_*_all.snap
```

It's not strictly required but I really prefer when tests, individually, clean up after themselves.